fix_input.py

```
@@ -0,0 +1,18 @@
+import re
+
+file_path = "crates/logfwd-runtime/src/pipeline/input_build.rs"
+with open(file_path, "r") as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)